policyengine_us/parameters/gov/states/md/tax/income/credits/senior_tax/amount/widow.yaml

```diff
@@ -0,0 +1,15 @@
+description: Maryland provides this senior tax credit amount for widow(er) filers.
+values: 
+  2021-01-01: 0
```

Comment:
I don't think we need this here, since you'll end up adding the credit to a list, so it won't be executed pre-2022 to break stuff in testing. We'll be a bit inconsistent here though under current implementations with some parameters needing historical parameters pre-implementation to avoid breaking. @nikhilwoodruff curious what you think.